resources/idlharness.js

```diff
@@ -637,11 +637,17 @@ function exposure_set(object, default_set) {
     if (exposed && exposed.length) {
         const { rhs } = exposed[0];
         // Could be a list or a string.
-        const set = rhs.type === "identifier-list" ?
+        const set =
+            rhs.type === "*" ?
```

Comment:
There's an initial list of spec PRs here: https://github.com/tc39/proposal-shadowrealm/issues/331. We haven't seen a use case for exposing things in ShadowRealms but not on some other globals, because those other globals could then create a ShadowRealm and use the API from there.